RawOpsGenerated.swift

```diff
@@ -2130,7 +1451,7 @@ public static func avgPool<T: FloatingPoint & TensorFlowScalar>(
   ksize: [Int32],
   strides: [Int32],
   padding: Padding,
-  dataFormat: DataFormat = .nhwc
+  dataFormat: DataFormat = .b'nhwc'
```

Comment:
@eaplatanios  This is malformed and causing https://github.com/apple/swift/pull/23700 to fail. I'm going to revert this PR for now.

Reply:
Oh I know why this happens. It’s a Python 3 compatibility bug. I’ll fix it once I get to my laptop.

Reply:
Fixed in #19 .